### DISCUSSION TITLE
examples/ui/text.rs doesnt run

### DISCUSSION THREAD

**Discussion by ageofcodes** [bevyengine/bevy#11300] [Q&A]
examples/ui/text.rs doesnt run correctly on bevy = "0.12.1"

**Comment by mockersf** [comment]
If you took the example from git, you need to change to the branch of the release (https://github.com/bevyengine/bevy/blob/release-0.12.1/examples/ui/text.rs) or `latest` (https://github.com/bevyengine/bevy/blob/latest/examples/ui/text.rs).

The `main` branch is evolving, and preparing for the next release of Bevy
